__tests__/components/Feed.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { shallow } from '../../enzyme';
```

Comment:
please, try not to use shallow, use mount instead

Reply:
when shallow would be preferible?

Reply:
never, hehe
https://kentcdodds.com/blog/why-i-never-use-shallow-rendering/